### PR TITLE
CFE-921: Enable GCPLabelsTags featureGate by default for GA

### PR DESCRIPTION
**Pull request by bharath-b-rh**
Disabling feature gates checks for configuring GCP labels and tags to make it GA.

### CODE REVIEW

File: features/features.go [hunk 1]
```diff
@@ -168,13 +168,6 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
-	FeatureGateGCPLabelsTags = newFeatureGate("GCPLabelsTags").
-					reportProblemsToJiraComponent("Installer").
-					contactPerson("bhb").
-					productScope(ocpSpecific).
-					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```
Comment (JoelSpeed) [inline features/features.go:175]: To promote a feature to GA, it is expected first that you add the default feature set here, and then clean up the gating in a later release

**Review by JoelSpeed — COMMENTED** [review]
You need to just promote the gate in the first instance, and then remove the gating in a later release

File: features/features.go (hunk 1 above)
Comment (bharath-b-rh) [reply features/features.go:175]: @JoelSpeed Thank you reviewing the changes!
I have made the required changes. PTAL.

**Comment by bharath-b-rh** [comment]
@JoelSpeed verify CI job is failing with below error, but there are tests present [here](https://github.com/openshift/api/blob/master/config/v1/tests/infrastructures.config.openshift.io/GCPLabelsTags.yaml) for the feature. Please let me know if different set of test cases need to be added.
```
Query component readiness for all test run results for feature gate "GCPLabelsTags" on clusterProfile ["Hypershift" "SelfManagedHA"]
Query component readiness for all test run results for pattern "FeatureGate:GCPLabelsTags]" on variant main.JobVariant{Cloud:"gcp", Architecture:"amd64", Topology:"ha"}
INSUFFICIENT CI testing for "GCPLabelsTags".
F0805 11:04:34.173020  169621 root.go:64] Error running codegen: error: only 0 tests found, need at least 5 for "GCPLabelsTags" on {gcp amd64 ha}
make: *** [Makefile:56: verify-scripts] Error 255
```

**Comment by JoelSpeed** [comment]
@bharath-b-rh The test here is looking for E2E testing, do you have any E2E testing in the origin test suite? Have QE worked on any E2E tests?

**Comment by jianli-wei** [comment]
> @bharath-b-rh The test here is looking for E2E testing, do you have any E2E testing in the origin test suite? Have QE worked on any E2E tests?

FYI the existing E2E tests in PROW CI, for GCP user-defined labels & tags: 
```
[jiwei@jiwei-thinkpadt14gen4 release]$ grep -E "as: gcp-ipi-.*tags" ci-operator/config/ -rl
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml
[jiwei@jiwei-thinkpadt14gen4 release]$ grep -E "as: gcp-ipi-.*tags" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml -n -A14
1909:- as: gcp-ipi-labels-tags-filestore-csi-fips-tp-amd-f28-destructive
1910-  cron: 32 16 7 * *
1911-  steps:
1912-    cluster_profile: gcp-qe
1913-    dependencies:
1914-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
1915-    dependency_overrides:
1916-      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.17
1917-    env:
1918-      FEATURE_SET: TechPreviewNoUpgrade
1919-      FIPS_ENABLED: "true"
1920-    test:
1921-    - chain: openshift-e2e-test-qe-destructive
1922-    workflow: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-filestore-csi
1923:- as: gcp-ipi-user-labels-tags-filestore-csi-tp-arm-f14
1924-  cron: 13 9 10,26 * *
1925-  steps:
1926-    cluster_profile: gcp-qe
1927-    dependencies:
1928-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
1929-    dependency_overrides:
1930-      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.17
1931-    env:
1932-      COMPUTE_NODE_TYPE: t2a-standard-4
1933-      FEATURE_SET: TechPreviewNoUpgrade
1934-      OCP_ARCH: arm64
1935-    test:
1936-    - chain: openshift-e2e-test-qe
1937-    workflow: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-filestore-csi
[jiwei@jiwei-thinkpadt14gen4 release]$

**Comment by JoelSpeed** [comment]
@jianli-wei Is that you saying that QE approve the promotion of this feature? Are you happy that the feature meets its goals and has sufficient regression testing in place?

**Comment by jianli-wei** [comment]
> @jianli-wei Is that you saying that QE approve the promotion of this feature? Are you happy that the feature meets its goals and has sufficient regression testing in place?

@JoelSpeed Yes, our existing PROW CI tests have the E2E tests enabled already, and I will update [my PR](https://github.com/openshift/release/pull/48249) for post-installation check of user-defined labels & tags after the feature is ready. Thanks!

**Comment by bharath-b-rh** [comment]
@JoelSpeed ^^

**Pull request merged**